e2e/hardhatTransactionFlowSwaps.spec.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable no-undef */
 /* eslint-disable jest/expect-expect */
-import { exec } from 'child_process';
 import { Contract } from '@ethersproject/contracts';
```

Comment:
i'm running e2e locally and i see this emoji on top left? 
![image](https://user-images.githubusercontent.com/12115171/191334379-69400eed-c750-440c-8e2a-2dfde48b3b09.png)

Reply:
![Uploading image.png…]()

Reply:
ah! yes! this is for faster connecting with hardhat!

Reply:
ah i see it `dev-button-hardhat`